_posts/2017/2017-06-27-why-a-techwriter-should-write-your-api-docs.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "API writing - just copy and paste?"
+date: "2017-06-27 06:34:02"
+image: blog-header/api-journey.jpg
+categories: api
+authors: ["Birgit"]
+---
+
+Can you imagine that?
```

Comment:
The first sentence might be perceived a bit offensive. 
Maybe, you can weaken the harsh tone like this: 

A short while ago, it came to my notice that some people think that API writing is just about copying and pasting text. Or that API writers just make things nice and shiny; polish some wording here and there. I'm a API writer and will uncover the truth of my work in this blog post.